app/src/ui/organism/product-category/ProductCategory.js

```diff
@@ -87,7 +80,7 @@ const ProductCategory = () => {
         <div className="es-pt-6">
           <div className="row gy-4">
             <div className="col-lg-4 col-md-6">
-              <div className="es-product-wrapper">
+              <div className="es-product-wrapper pb-4">
                 <div className="ps-4 pt-4">
```

Comment:
we can remove this div - just change in parent div line 83 pb-4 to p-4 
this way we will padding in all directions so padding start ps-4 and padding top pt-4 is unnecessary

Reply:
className pb-4 is necessary because this is padding for parent and first child box with text and button needs ps-4 and pt-4 second child img don't needs